vignettes/profiling.Rmd

```diff
@@ -0,0 +1,68 @@
+---
+title: "Profiling Performance"
+author: "Thomas Lin Pedersen"
+date: "`r Sys.Date()`"
```

Comment:
Can you delete date? It's not really that useful (I need to remove it from the template)